### PR TITLE
Admin panel blob storage

### PR DESCRIPTION
**Pull request by Kasterov**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: Streetcode/Streetcode.BLL/DTO/Media/AudioDTO.cs [hunk 1]
```diff
@@ -6,6 +6,7 @@ public class AudioDTO
 {
     public int Id { get; set; }
     public string? Description { get; set; }
-    public UrlDTO Url { get; set; }
+    public string BlobName { get; set; }
+
```
Comment (oshut) [inline Streetcode/Streetcode.BLL/DTO/Media/AudioDTO.cs:10]: redundant line

File: Streetcode/Streetcode.BLL/MediatR/Media/Audio/Create/CreateAudioHandler.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            request.Audio.Title,
+            request.Audio.Extension);
+
+        var audio = _mapper.Map<DAL.Entities.Media.Audio>(request.Audio);
+
+        audio.BlobName = $"{hashBlobStorageName}.{request.Audio.Extension}";
+
+        await _repositoryWrapper.AudioRepository.CreateAsync(audio);
+
+        var resultIsSuccess = await _repositoryWrapper.SaveChangesAsync() > 0;
```
Comment (oshut) [inline Streetcode/Streetcode.BLL/MediatR/Media/Audio/Create/CreateAudioHandler.cs:39]: don't you think that `isResultSuccess `or `isSavingSucceeded` or even just `isSuccess` would be more appropriate than current name?

**Review by Tysyatsky — APPROVED** [review]
Great job pal! Waiting for the meeting with a guide how to work with that in future. Thank you.

**Review by MementoMorj — COMMENTED** [review]
OMG111AWS!!!!GOAT!!!!

**Review by Katerix — APPROVED** [review]
awesome! useful and fun blob storage)

**Pull request merged**